scripts/ci/ovnkube-setup.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+set -eu
+
+# DEPENDENCIES:
+# This scripts assumes the following packages are installed:
+#   docker
+#   go
+#   python and pip
+# PERMISSIONS:
+# This script can be executed without root privilegeds. 'sudo' will be used
+# for the specific tasks that do require it
+
+# Global env vars that modify the behavior of the script:
+KIND_VERSION=${KIND_VERSION:-v0.10.0}
+K8S_VERSION=${K8S_VERSION:-v1.20.0}
+OVNK8S_VERSION=${OVNK8S_VERSION:-ab8bcb41ae215afd3890f243fd99e1d0cce5feed}
+OVNK8S_REPO=https://github.com/ovn-org/ovn-kubernetes.git
+
+# Global vars
+WORKDIR=${HOME}/ovnkube-setup
```

Comment:
Could we use $WORKSPACE if its defined and use $HOME/ovnkube-setup otherwise ?
That would allow 2 things : rely on Jenkins to clean everything and reuse the existing GOPATH (and therefore the cache) to speed up build

Reply:
Sure, done!